test/e2e/page-objects/pages/header-navbar.ts

```diff
@@ -35,6 +38,12 @@ class HeaderNavbar {
     await this.driver.clickElement(this.accountMenuButton);
   }
 
+  async openSnapListPagee(): Promise<void> {
```

Comment:
minor nit : openSnapListPagee to openSnapListPage

Reply:
Great catch, thanks ! corrected in [5b28d64](https://github.com/MetaMask/metamask-extension/pull/27327/commits/5b28d6440a68e646326cf52e917d5e547bcff0fb)